src/server/maps/MapleReactorFactory.java

```diff
@@ -38,6 +38,61 @@
     private static MapleDataProvider data = MapleDataProviderFactory.getDataProvider(new File(System.getProperty("wzpath") + "/Reactor.wz"));
     private static Map<Integer, MapleReactorStats> reactorStats = new HashMap<Integer, MapleReactorStats>();
 
+    
+    public static final MapleReactorStats getReactorS(int rid) {
```

Comment:
Why are there 2 similarly named functions for getReactor?

Reply:
Afaik heavenMS uses getRectorS to load in the MCPQ reactors. The old reactor loading function is unchanged.